From 916cee7bb2664029e16633af4fac4915de28049c Mon Sep 17 00:00:00 2001
From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Tue, 6 Sep 2011 07:37:26 +0200
Subject: [PATCH 18/18] Use symbolic names for FolderSeparator

Some .ini parsers have trouble with parsing

  FolderSeparator=\

So let's use symbolic names "DOS"/"Unix" instead.
---
 README                               |    4 ++--
 media-players/sandisk_sansa-fuze.mpi |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/README b/README
index c3d9333..c3c5c60 100644
--- a/README
+++ b/README
@@ -76,8 +76,8 @@ or file a bug).
 - Formats: a semi-colon-separated list of the mime-types of the
   playlist formats supported by the device
 - FolderSeparator: separator between directory/file names in playlist files. If
-  not given, '/' is assumed (standard Unix separator). Some players require the
-  DOS separator '\'.
+  not given, "Unix" is assumed, which uses '/'. Some players require 
+  "DOS", which will use '\'. Other values are invalid and will be ignored.
 - LineEnding: line separator in playlist files. This can have the values 'CR',
   'LF', or 'CRLF', which correspond to 0x0D (\r), 0x0A (\n), and 0x0D0A (\r\n)
   respectively. If not given, the default behaviour depends on the music player
diff --git a/media-players/sandisk_sansa-fuze.mpi b/media-players/sandisk_sansa-fuze.mpi
index 3d942b7..6a07bb5 100644
--- a/media-players/sandisk_sansa-fuze.mpi
+++ b/media-players/sandisk_sansa-fuze.mpi
@@ -10,7 +10,7 @@ OutputFormats=audio/mpeg;audio/x-ms-wma;audio/audible;application/ogg;audio/ogg;
 
 [Playlist]
 Formats=audio/x-mpegurl
-FolderSeparator=\
+FolderSeparator=DOS
 
 [storage]
 RequiresEject=true
-- 
1.7.6

